EIPS/eip-2309.md

```diff
@@ -46,6 +45,10 @@ The address executing the transaction **MUST** own all the tokens within the ran
 
 The `fromTokenId` and `toTokenId` **MUST** be a consecutive range of tokens IDs.
 
+The `fromTokenId`, `fromAddress`, and `toAddress` **MUST** be indexed parameters
+
+The `toTokenId` **MUST NOT** be an indexed parameter
```

Comment:
Nitpicking:
```suggestion
The `toTokenId` **MUST NOT** be an indexed parameter.
```

Given the rest of the statements were terminated with a period.